### PR TITLE
Syncing Summary for Git (Issue #1003)

### PR DESCRIPTION
**Pull request by njanne19**
[Issue link](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/1003)
[Rawgit](https://rawgit.com/njanne19/njanne19.github.io/beta-branch/#!pages/gitandrepositories.md)

__Checklist__

- [x] Simplified statements I had in original issue
- [x] Narrowed the instructions down to only a few steps to achieve desired simplicity
- [x] Added links to the documentation of git calls I mentioned

### CODE REVIEW

**Comment by njanne19** [comment]
@sarah-lu102 I can do that right now. Thanks for the comment

**Comment by njanne19** [comment]
@sarah-lu102 
I fixed the issues you were talking about, I'll also send a message @dogi in the gitter to make sure he sees this

File: pages/gitandrepositories.md [hunk 1]
```diff
@@ -143,7 +143,8 @@ For more info on differences of merging vs. rebasing (and when to use which one)
 ## Summary of steps
 1. Clone your git \<username\> repository and open your terminal window to it by using ``cd``
 
-#### Note that for steps 2 and 3, developers should do this every time they begin to work. This way we as a team can minimize data loss, and can save you some time. You want to make sure that you are always up to date with the github.
+NOTE: [for steps 2 and 3, developers should do this every time they begin to work. This way we as a team can minimize data loss, and can save you some time. You want to make sure that you are always up to date with the github.]
```
Comment (kylemathias) [inline pages/gitandrepositories.md]: take out '[' and ']'. Its already a note, it dose not need to be parenthesized. Capitalize the first word in the sentence. change github to GitHub.

Be careful when you are changing a sentence that you are also naturally fixing the grammar.

Comment (njanne19) [inline pages/gitandrepositories.md]: @kylemathias Just updated what you said

**Pull request merged**